src/Discord.Net.Interactions/Attributes/EnumChoiceAttribute.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+
+namespace Discord.Interactions
+{
+    /// <summary>
+    ///     Customize the displayed value of a slash command choice enum.
+    /// </summary>
```

Comment:
Can you add a remark here mentioning that this attribute only works with the default enum TypeConverter and not with custom ones.